Library/src/Systems/ECommerce/ECommerceSystem.cpp

```diff
@@ -56,11 +56,11 @@ void RemoveUrl(csp::common::String& Url)
 namespace csp::systems
 {
 
-ECommerceSystem::ECommerceSystem() : SystemBase(), ShopifyAPI(nullptr)
+ECommerceSystem::ECommerceSystem() : SystemBase(nullptr, nullptr), ShopifyAPI(nullptr)
 {
 }
 
-ECommerceSystem::ECommerceSystem(csp::web::WebClient* InWebClient) : SystemBase(InWebClient)
+ECommerceSystem::ECommerceSystem(csp::web::WebClient* InWebClient) : SystemBase(InWebClient, nullptr)
```

Comment:
I am aware that @MAG-ChristopherAtkinson isn't keen on setting the EventBus pointer to nullptr, however we would have to modify the 10 classes that currently derive from SystemBase and are not currently using the EventBus. 
I am not quite sure what the ramifications of such a change would be -- I am happy to do it if people think it is worth doing.

Reply:
I created a story for this (OF-1464)